### ISSUE TITLE
saving the intersected shapefile failed for netcdf format of 0 to 360 and shapefile with lon values of higher than 180

### ISSUE THREAD

**Issue by ShervanGharari** [ShervanGharari/EASYMORE#23]
```File "/home/gut428/.local/lib/python3.8/site-packages/easymore/easymore.py", line 151, in nc_remapper
    shp_int.to_file(self.temp_dir+self.case_name+'_intersected_shapefile.shp') # save the intersected files
  File "/home/gut428/.local/lib/python3.8/site-packages/geopandas/geodataframe.py", line 1086, in to_file
    _to_file(self, filename, driver, schema, index, **kwargs)
  File "/home/gut428/.local/lib/python3.8/site-packages/geopandas/io/file.py", line 330, in _to_file
    colxn.writerecords(df.iterfeatures())
  File "/home/gut428/.local/lib/python3.8/site-packages/fiona/collection.py", line 342, in writerecords
    self.session.writerecs(records, self)
  File "fiona/ogrext.pyx", line 1198, in fiona.ogrext.WritingSession.writerecs
RuntimeError: Failed to write record:
```
saving the intersected shapefile for North Asia failed.
In this issue there is a netcdf file with format 0 to 360 and a shapefile (merit hydro basin) with longitude more than 180 degrees. It should be tested were the issue of failure in saving is coming from. more test are needed.

**Comment by ShervanGharari** [comment]
is now taken care in `develop` branch and will be merge with `master` after testing

**Comment by ShervanGharari** [comment]
the code have been tested and work fine for shapefile that are in format of 0 to 360 for their lon values and pass the lon of 180. example is provided in Untitled Jupiter notebook under example folder.

**Comment by ShervanGharari** [comment]
is fixed in easymore version 0.0.2

**Comment by wknoben** [comment]
I run into this exact issue (NorthAsia shapefile) on Copernicus with `easymore     0.0.2+computecanada`. Trying to figure out what's going on..
